addons/web/static/src/less/form_view_extra.less

```diff
@@ -176,6 +176,9 @@
     .o_form_view {
         .o_form_sheet_bg, .o_form_sheet {
             border: none;
+            &.o_form_sheet {
+                min-height: initial;
+            }
```

Comment:
Use `min-height: 0` which is more explicit and the rule should not be nested this way. `.o_form_sheet_bg.o_form_sheet` does not make sense I think? :wink: